space/space.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strings"
 )
 
+const SystemSpace = "system.space"
```

Comment:
I dislike having a space ID and still refer to the space by a name. Instead you could just treat the name a random string and replace

```go
const SystemSpace = "system.space"
```

with

````go
var SystemSpace = uuid.FromStringOrNil("2e0698d8-753e-4cef-bb7c-f027634824a2")
````

where `2e0698d8-753e-4cef-bb7c-f027634824a2` is a string that is not supposed to change just like `system.space` is not meant to be changed.

I'm doing the same for work item types at the moment and I think we should refer to things by their IDs and not by their names.

What do you think?